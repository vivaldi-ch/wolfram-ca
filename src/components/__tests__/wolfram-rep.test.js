import React from 'react';
import renderer from 'react-test-renderer';

import WolframRep from '../wolfram-rep';

const arr = [0, 1, 0, 1, 0, 1, 0, 1];

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<WolframRep wolframArr={arr} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
