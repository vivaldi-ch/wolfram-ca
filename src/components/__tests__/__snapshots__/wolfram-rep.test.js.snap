// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginBottom": "1em",
    }
  }
>
  <div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square white"
      />
      <div
        className="square white"
      />
      <div
        className="square white"
      />
    </div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square white"
      />
    </div>
  </div>
  <div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square white"
      />
      <div
        className="square white"
      />
      <div
        className="square black"
      />
    </div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square black"
      />
    </div>
  </div>
  <div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square white"
      />
      <div
        className="square black"
      />
      <div
        className="square white"
      />
    </div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square white"
      />
    </div>
  </div>
  <div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square white"
      />
      <div
        className="square black"
      />
      <div
        className="square black"
      />
    </div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square black"
      />
    </div>
  </div>
  <div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square black"
      />
      <div
        className="square white"
      />
      <div
        className="square white"
      />
    </div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square white"
      />
    </div>
  </div>
  <div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square black"
      />
      <div
        className="square white"
      />
      <div
        className="square black"
      />
    </div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square black"
      />
    </div>
  </div>
  <div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square black"
      />
      <div
        className="square black"
      />
      <div
        className="square white"
      />
    </div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square white"
      />
    </div>
  </div>
  <div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square black"
      />
      <div
        className="square black"
      />
      <div
        className="square black"
      />
    </div>
    <div
      className="squareRowWrapper"
    >
      <div
        className="square black"
      />
    </div>
  </div>
</div>
`;
